sci-mathematics/glucas/glucas-2.9.0.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="A client to make primality test for Mersenne numbers"
+DESCRIPTION="Glucas is related to GIMPS effort to search the biggest primes \ 
+ever found."
+HOMEPAGE="https://sourceforge.net/projects/glucas/"
+COPYRIGHT="2001 gbvalor"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/glucas/Glucas-$portVersion.tar.gz"
+CHECKSUM_SHA256="fde422220245788c0d8aeaec6a89f26e849905b1417c5e6ff1a01f56c595cadc"
+PATCHES="glucas-2.9.0.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+glucas = $portVersion
```

Comment:
Looks like you removed the tab(s) here:
```diff
-glucas = $portVersion
+	glucas = $portVersion
```
Only 1 tab (also in the other places, check them, you also got atleast 1 with a tab and a space, others with only spaces)